CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 #### New Features
 
+- Updated `Session` class to be thread-safe. This allows concurrent dataframe transformations, dataframe actions, UDF and store procedure registration, and concurrent file uploads.
+  - The feature is disabled by default and can be enabled by setting `PYTHON_SNOWPARK_ENABLE_THREAD_SAFE_SESSION` to `True` for account or in `session_parameter` when creation the `Session` object.
```

Comment:
is the session parameter approach explicitly documented somwhere, if yes, might be good to add a link for this, otherwise maybe not mention it here, it is not very obvious to me what does that mean here.
Also `PYTHON_SNOWPARK_ENABLE_THREAD_SAFE_SESSION` that is a parameter can only be enabled by snowflake right? maybe say concat snowflake to enable this flag? i am not sure what is the best way to doc the snowflake parameters with open source project here, maybe PM or doc can help here?

Reply:
its not documented right now. I'll just remove the mention updating is using `session_parameter`.